samples/qir/oracle-generator/qsharp/project.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.Quantum.Sdk/0.16.2105142209-alpha">
```

Comment:
Does this require an alpha version? Is it possible to use a released version?

Reply:
I am not sure, I'll try to use the version of the upcoming release.

Reply:
The package `Microsoft.Quantum.Qir.Runtime` does not exist in a released version yet. I suggest to wait until the May release and then change the version number, and keep this alpha version for now.

Reply:
Makes sense; just wanted to double-check on that. I think it's fine to keep the version number as-is, then, but I would emphasize in the README that QIR generation is still in preview to help set user expectations.